packages/mgt-components/src/components/mgt-todo/mgt-todo.scss

```
@@ -6,12 +6,131 @@
  */
 
 @import '../mgt-tasks-base/mgt-tasks-base.scss';
+@import '../mgt-tasks-base/mgt-tasks-base.theme.scss';
 
 :host {
   @extend :host;
   border-radius: 8px;
   .Task {
     @extend .Task;
-    border: none;
+    width: auto;
+    &.NewTask {
+      margin: 14px 0 0 0;
+    }
+    &.Complete {
+      text-decoration: line-through;
+      border: set-var(border, $theme-default, $tasks);
+      background: $task-background-color--completed;
+    }
   }
+
+  .TaskDetails {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 2px;
+    height: 34px;
+
+    border-radius: 4px;
+    .Task {
+      > div {
+        display: flex;
+        align-items: center;
+        width: 200px;
+      }
+    }
+    .title {
+      width: 600px;
+    }
+    .TaskDelete {
+      display: flex;
+    }
+    .TaskDue {
+      width: 111px;
+      height: 32px;
+      text-decoration: inherit;
+      display: flex;
+
+      .TaskCalendar {
+        display: flex;
+        margin-top: 5px;
+        margin-inline-end: 10px;
+
+        svg {
+          fill: $task-color;
+        }
+      }
+      .TaskDueDate {
+        display: flex;
+        margin-top: 5px;
+      }
+    }
+  }
+
+  fluent-text-field {
+    width: 800px;
```

This width is likely the cause of the "weirdness" 

```suggestion
    width: 100%;
```